.github/workflows/run_tests_internal.yml

```diff
@@ -67,4 +67,4 @@ jobs:
             FINAL_PYTEST_MARKER="${{ inputs.pytest_marker }} and not scheduled_only"
           fi
           python3 -m pip install -e . --no-dependencies &&
-          python3 -m pytest -v --pyargs MaxText.tests -m "${FINAL_PYTEST_MARKER}" --durations=0
+          python3 -m pytest -v -m "${FINAL_PYTEST_MARKER}" --durations=0
```

Comment:
Why is this change needed?

Reply:
The test directory got renamed and by default pytest is reading from `tests` directory so it doesn't need this explicit argument set. Furthermore the `testpaths` in `pytest.ini` explicitly specifies this `tests` directory.